### PR TITLE
Add text container

### PR DESCRIPTION
**Pull request by jtart**
_Creates a Text container components that data extraction out of an Optimo Text block - ported logic from old Text component. Creates a Text component that wraps the markdown-to-jsx library._

* [x] Part of https://github.com/bbc/simorgh/issues/246
* [x] Tests added for new features

* [ ] Test engineer approval

### CODE REVIEW

**Review by sareh — CHANGES_REQUESTED** [review]
Great work on splitting out the existing text component into a container and component. 

Just one suggestion - I think that the Markdown component needs to be renamed. This is because Markdown is just an implementation detail, in terms of what we get from our API and how it's parsed here.

The component name should be reflective of the User Interface. Perhaps `FormattedText` would be a suitable alternative.

**Review by sareh — APPROVED** [review]
Thanks for the changes. Looks great!

**Review by DemonusPC — APPROVED** [review]
Yep, looks great

**Review by pjlee11 — APPROVED** [review]

**Review by jp-bbc — APPROVED** [review]

File: src/app/containers/Text/index.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { textModelPropTypes } from '../../models/propTypes/text';
+import Text from '../../components/Text';
+
+const TextContainer = ({ blocks }) => {
+  if (!blocks) return null;
```
Comment (DemonusPC) [inline src/app/containers/Text/index.jsx]: Shouldn't this be in the container?

File: src/app/components/Text/index.jsx [hunk 2]
```diff
@@ -1,19 +1,24 @@
...
-} from '../../models/propTypes/text';
 
-const Text = ({ blocks }) =>
-  blocks.map(({ blockId, model }) => (
-    <Markdown key={blockId} options={{ forceBlock: true }}>
-      {model.text}
-    </Markdown>
-  ));
+const Text = ({ text, options }) => {
+  if (!text) return null;
```
Comment (DemonusPC) [inline src/app/components/Text/index.jsx:6]: In my opinion, this logic should be handled in the container

**Review by DemonusPC — CHANGES_REQUESTED** [review]

File: src/app/components/Text/index.jsx (hunk 2 above)
Comment (jtart) [reply src/app/components/Text/index.jsx:6]: Agreed, won't do that in this PR though.

**Pull request merged**